lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -97,6 +107,35 @@ public function __construct(mysqli $conn, $prepareString)
         $this->_bindedValues = array_fill(1, $paramCount, null);
     }
 
+    /**
+     * Converts named placeholders (":parameter") into positional ones ("?"), as MySQL does not support them.
+     *
+     * @param string $query The query string to create a prepared statement of.
+     *
+     * @return string
+     */
+    private function convertNamedToPositionalPlaceholders($query)
```

Comment:
Type declarations of docblocks please
```suggestion
     */
    private function convertNamedToPositionalPlaceholders(string $query) : string
```